src/runtime/contrib/tensorrt/tensorrt_logger.h

```diff
@@ -39,7 +39,7 @@ class TensorRTLogger : public nvinfer1::ILogger {
  public:
   TensorRTLogger() : TensorRTLogger(Severity::kWARNING) {}
   explicit TensorRTLogger(Severity severity) : reportable_severity(severity) {}
-  void log(Severity severity, const char* msg) override {
+  void log(Severity severity, const char* msg) noexcept override {
```

Comment:
Is this change required to build against TRT 8? Could you check we can still compile against earlier TRT versions such as 7 even with this change? If we can't, then we should add `#if` around it.

Reply:
Yes, this is required for TRT8. I've tested TRT7 and it works. In TRT 8 the signature changes to`virtual void log(Severity severity, AsciiChar const* msg) noexcept = 0;` from `virtual void log(Severity severity, const char* msg) TRTNOEXCEPT = 0;` in TRT7.